README.md

```diff
@@ -24,9 +24,9 @@ We have a fantastic community growing on Discord. Click [here](https://discord.g
 4. Install dependencies with:
 
 ```bash
-npm install
+npm install && npm audit fix --force
```

Comment:
Hey Edward,
We dont want to add "run audit fix" to the readme, since it will update packages.

Ideally this will be done as a separate PR, and the changes will be pushed to everyone. 
I ran this locally myself during the stream, and it is updating sentry, so we will need to do a little investigating on this first. Thanks

Reply:
Thank you JP, this is a good point! I will omit this from the final doc